src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/InputSkipIfLastAndSharedWithCase.java

```diff
@@ -0,0 +1,113 @@
+package com.puppycrawl.tools.checkstyle.checks.coding;
+
+
+public class InputSkipIfLastAndSharedWithCase
+{
+    void method(int i) {
+        switch (i) {
+            case 1:
+            default: // No violation with the new option is expected
+                break;
+            case 2:
+                break;
+        }
+
+        switch (i) {
+            case 1:
+            default: // No violation with the new option is expected
+            case 2:
+                break;
+            case 3:
+                break;
+        }
+
+        switch (i) {
+            default: // No violation with the new option is expected
```

Comment:
i also want clarification on that because i think we should allow anywhere as long as surrounded by case